js/models/purchase/Order.js

```diff
@@ -34,9 +34,9 @@ export default class extends BaseModel {
 
   addAddress(sAddr) {
     // this will convert and set an address from the settings
-    const shipTo = sAddr.get('name');
-    const address =
-      `${sAddr.get('addressLineOne')} ${sAddr.get('addressLineTwo')} ${sAddr.get('company')}`;
+    const company = sAddr.get('company');
+    const shipTo = `${sAddr.get('name')}${company ? `, ${company}` : ''}`;
+    const address = `${sAddr.get('addressLineOne')} ${sAddr.get('addressLineTwo')}`;
```

Comment:
For some reason, the previous change to this broke the purchase call on the server, so I moved the company to be appended to the send to value, which is probably more correct.